modules/eco-node-health-check-operator-creating-node-health-check.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-health-check-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-node-health-check-operator-creating-node-health-check_{context}"]
+= Creating a node health check
+Using the web console, you can create a node health check to identify unhealthy nodes and specify the remediation type and strategy to fix them.
+
+.Procedure
+
+. From the *Administrator* perspective of the {product-title} web console, click *Compute* -> *NodeHealthChecks* -> *CreateNodeHealthCheck*.
+. Specify whether to configure the node health check using the *Form view* (the default setting) or the *YAML view*.
+. Enter a *Name* for the node health check. The name must consist of lower case, alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
+. Specify the *Remediator* type, *Self node remediation* or *Other*, and the *Remediation strategy*. The Self node remediation strategy applies only to the Self Node Remediation operator. The Self node remediation options are part of the the Self Node Remediation operator that is installed with the Node Health Check operator. Selecting *Other* requires an *API version*, *Kind*, *Name*, and *Namespace* to be entered from an existing resource of a remediator.
+. Make a *Nodes* selection by specifying labels of the nodes you want to remediate. The selection matches labels that you want to check. If more than one label is specified, the nodes must contain each label. The default value is empty, which selects all nodes.
+. Specify the minimum number of healthy nodes (in percentage or number) required for a *NodeHealthCheck* to remediate nodes in the targeted pool. If the number of healthy nodes equals to or exceeds the limit set by *Min healthy*, remediation occurs. The default value is 51%.
+. Specify a list of *Unhealthy conditions* that if a node meets determines whether the node is considered unhealthy, and requires remediation. You can specify the *Type*, *Status* and *Duration*. You can also create your own custom type.
+. Click *Create* to create the node health check.
+
+.Verification
+
+* Navigate to the *Compute* -> *NodeHealthCheck* page and verify that the corresponding node health check is listed. Once created, node health checks can be paused, modified, and deleted.
```

Comment:
It may be worth to mention that the status field displays the NHC current status and status of successfully created NHC is Enabled

Reply:
> status of successfully created NHC is Enabled

as soon as you see the NHC resource, ir was successfully created. The status isn't related to creation. Or in other words: a successfully created NHC can also be disabled.

Reply:
Thanks @anna-savina @slintes. I have updated the Verification step to include that the status of the node health check is displayed.